dev/breeze/src/airflow_breeze/commands/developer_commands.py

```diff
@@ -515,6 +515,9 @@ def static_checks(
         force_build=force_build,
         image_tag=image_tag,
         github_repository=github_repository,
+        # for static checks we do not want to regenerate dependencies before pre-commits are run
+        # we want the pre-commit to do it for us (and fail in case the dependencies are updated)
```

Comment:
```suggestion
        # we want the pre-commit to do it for us (and raise failures if any only when dependencies are updated)
```
otherwise, it may sound like it will always fail when dependencies are updated? 🤔